doc/src/selections.rst

```diff
@@ -83,9 +95,14 @@ equivalent to ``atoms: name == H``.
 Then, if no context is given, the ``atoms`` context is used. This make ``atoms:
 name == H`` equivalent to ``name == H``.
 
-And finally, the ``==`` comparison operator is the default one if no operator is
+Then, the ``==`` comparison operator is the default one if no operator is
```

Comment:
May I suggest:

"If no comparison operator is given, the ``==`` operator is used by default."

Reply:
Also, does this mean I can use other relations like `>`? How does this work for strings?